src/OMSimulatorLib/System.h

```diff
@@ -174,6 +174,7 @@ namespace oms
 
     DirectedGraph initialUnknownsGraph;
     DirectedGraph outputsGraph;
+    DirectedGraph continuousTimeModeGraph; ///< filtered dependency graph, with connections of type Real;
```

Comment:
I think we should rename those graphs (but not in the components). 

```
DirectedGraph initializationGraph;
DirectedGraph simulationGraph;
DirectedGraph eventGraph;
```

Reply:
fixed with https://github.com/OpenModelica/OMSimulator/pull/829/commits/2d16fbadbd82059679c14106c3cd9ca0ccc71a83